public/app/features/explore/utils/ResultProcessor.ts

```diff
@@ -97,6 +99,11 @@ export class ResultProcessor {
     return data;
   }
 
+  deduplicateLogRows = (rows: LogRowModel[]) => {
+    // Deduplicate log rows with the same log uid
+    return uniqBy(rows, 'uid');
+  };
+
```

Comment:
@aocenas I have moved `deduplicateLogRows` to ResultProcessor and used lodash `uniqBy`.